DNN Platform/Connectors/GoogleAnalytics/GoogleAnalyticsConnector.cs

```diff
@@ -189,7 +189,7 @@ public bool SaveConfig(int portalId, IDictionary<string, string> values, ref boo
                 }
                 else
                 {
-                    trackingID = values["TrackingID"] != null ? values["TrackingID"].ToLowerInvariant().Trim() : string.Empty;
+                    trackingID = values["TrackingID"] != null ? values["TrackingID"].ToUpperInvariant().Trim() : string.Empty;
```

Comment:
```suggestion
                    trackingID = values["TrackingID"] != null ? values["TrackingID"].Trim() : string.Empty;
```

Should this just allow for whatever casing is used?  Is there a reason to upper-case it?

Reply:
@david-poindexter I see your comment on #3318, which seems to say that the code _should_ allow any case to be saved, rather than forcing uppercase.  Am I missing something?

Reply:
Sorry for my clunky wording @bdukes - that's what I getting for rushing.  😆 

It is CURRENTLY respecting case and saving the TrackingID value just as entered by the admin.  However, this is causing a problem with Google Analytics functionality.  It needs to be UPPERCASE in order to work as expected.